src/utils.ts

```diff
@@ -9,10 +9,18 @@ import {
 } from '@alephium/web3'
 import { CLAMM, Invariant, InvariantInstance, Reserve, Utils } from '../artifacts/ts'
 import { TokenFaucet } from '../artifacts/ts/TokenFaucet'
-import { FeeTier, Pool, PoolKey, Position, Tick } from '../artifacts/ts/types'
+import { FeeTier, FeeTiers, Pool, PoolKey, Position, Tick } from '../artifacts/ts/types'
+import { FeeTierSlots } from './consts'
 
 export const MAP_ENTRY_DEPOSIT = ONE_ALPH / 10n
 
+export const DEFAULT_FEE_TIERS: FeeTiers = {
```

Comment:
Maybe `EMPTY_FEE_TIERS`? DEFAULT in this context could imply we already push data of some tiers we plan to use.